plugins/crowdstrike_falcon_intelligence/icon_crowdstrike_falcon_intelligence/actions/getSubmissionsIDs/action.py

```diff
@@ -0,0 +1,26 @@
+import insightconnect_plugin_runtime
+from .schema import GetSubmissionsIDsInput, GetSubmissionsIDsOutput, Input, Output, Component
+
+# Custom imports below
+
+
+class GetSubmissionsIDs(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="getSubmissionsIDs",
+            description=Component.DESCRIPTION,
+            input=GetSubmissionsIDsInput(),
+            output=GetSubmissionsIDsOutput(),
+        )
+
+    def run(self, params: dict = None):
+        limit = params.get(Input.LIMIT, 5000)
+        if limit > 5000:
+            self.logger.warning(
+                "Provided limit value is larger than 5000 but this action will return up to 5000 results."
+            )
```

Comment:
I think that way of handling the limit could potentially confuse the customer. Maybe we should throw an error, or use some paging if its available in their API?